Airports/EGSS/SMR/Labels.txt

```diff
@@ -29,12 +29,22 @@
 "V1" N051.53.02.900 E000.13.52.000 standHold
 "V3" N051.52.59.800 E000.13.47.000 standHold
 ;Stands
-"1" N051.53.10.640 E000.15.11.020 standHold
-"2" N051.53.12.610 E000.15.07.590 standHold
+N051.53.16.050 E000.15.04.512
+N051.53.16.548 E000.15.03.620
+N051.53.16.797 E000.15.03.190
+N051.53.17.085 E000.15.02.727
+N051.53.18.646 E000.14.59.954
```

Comment:
These lines aren't needed/useful in this section

Reply:
Oops, sorry. I was meant to delete them as I was overwriting the old coordinates